libraries/nanoFramework.Modbus/README.md

```diff
@@ -0,0 +1,5 @@
+# nanoFramework.Modbus
+Serial port Modbus-RTU protocol implementation.
+
+Repository: [https://github.com/yureng/nanoFramework.Modbus](https://github.com/yureng/nanoFramework.Modbus).<br>
```

Comment:
Markdown 101: if you need a new line, make the new line, don't use HTML code for that
```suggestion
Repository: [https://github.com/yureng/nanoFramework.Modbus](https://github.com/yureng/nanoFramework.Modbus).

```